src/lib/controllers/configuration-manager/index.d.ts

```diff
@@ -1,3 +1,18 @@
+/*
+Copyright 2019 Balena Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
 import { ValidationError } from 'jsonschema';
```

Comment:
Give this a separate name rather than using `index.d.ts`, as that implies it defines *all* the external types for the `config-manager` classes, which it doesn't.